catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/writer/CswRecordCollectionMessageBodyWriter.java

```diff
@@ -97,10 +99,13 @@ public void writeTo(CswRecordCollection recordCollection, Class<?> type, Type ge
         } else if (recordCollection.getOutputSchema()
                 .equals(OCTET_STREAM_OUTPUT_SCHEMA)) {
             Resource resource = recordCollection.getResource();
-            httpHeaders.put("Content-Type", Arrays.asList(resource.getMimeType()));
-            httpHeaders.put("Content-Disposition",
-                    Arrays.asList("inline; filename=\"" + resource.getName() + "\""));
-            IOUtils.copy(resource.getInputStream(), outStream);
+            httpHeaders.put(HttpHeaders.CONTENT_TYPE, Arrays.asList(resource.getMimeType()));
+            httpHeaders.put(HttpHeaders.CONTENT_DISPOSITION,
+                    Arrays.asList(String.format("inline; filename=%s", resource.getName())));
```

Comment:
It doesn't seem to make a difference. The value of the Content-Disposition is:
inline; filename="foobar", whether the quotes are added or not.